client/src/app/+about/about.component.html

```diff
@@ -1,3 +1,12 @@
-<p>
-  about works!
-</p>
+
+    <h3> אודות</h3>
```

Comment:
@eyalcohen4 The title / logo will be `h1` so this should be `h2` (we shouldn't skip `h2` and go to `h3` because of it's size. There should be a logical hierarchy, and we can after it style it in the `css`).